api/BroadcastChannel.json

```diff
@@ -435,10 +434,10 @@
               "version_added": "44"
             },
             "safari": {
-              "version_added": "preview"
+              "version_added": "15.4"
             },
             "safari_ios": {
-              "version_added": false,
+              "version_added": "15.4",
               "notes": "See <a href='https://webkit.org/b/171216'>bug 171216</a>."
```

Comment:
https://webkit.org/b/171216 is still open. Is it really supported or is the bug wrong?